### PR TITLE
add jQuery-tagEditor@1.0.20 w/ git auto-update

### LINKED ISSUE #8532: [Request] Add jQuery-tagEditor
**Library name:** jQuery-tagEditor
**Git repository url:**https://github.com/Pixabay/jQuery-tagEditor

### PR DESCRIPTION
**Pull request by nolsherry**
@pvnr0082t , please help me review this PR, thanks!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Pixabay/jQuery-tagEditor
 * Official website (optional, not the repository): https://goodies.pixabay.com/jquery/tag-editor/demo.html
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 245
   - Count of watchers: 21
   - Count of forks: 86
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

close #8532, cc @SimonSteinberger

### CODE REVIEW

File: ajax/libs/tag-editor/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "tag-editor",
+  "filename": "jquery.tag-editor.min.js",
+  "title": "tagEditor",
```
Comment (pvnr0082t) [inline ajax/libs/tag-editor/package.json]: We don't need `title` field. Please remove it.

**Comment by pvnr0082t** [comment]
@nolsherry Please minify the `css` file.

File: ajax/libs/tag-editor/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/tag-editor/package.json]: This can be reserve, no problem here.

**Comment by nolsherry** [comment]
@pvnr0082t, I have  Please minify the css file, please help me review this PR, thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Pull request merged**